packages/wrangler/src/__tests__/helpers/mock-upload-worker.ts

```diff
@@ -44,6 +45,9 @@ export function mockUploadWorkerRequest(
 ) {
 	const handleUpload: HttpResponseResolver = async ({ params, request }) => {
 		const url = new URL(request.url);
+		expect(url.hostname).toMatch(
+			options.expectedBaseUrl ?? "api.cloudflare.com"
```

Comment:
move the default to the params instead?

Reply:
Do you mean add it to the `options.expectedBaseUrl` parameter?
That would be pretty messy, since currently the parameter in the function is just `options`, so we would have to destructure that into all its properties in order to set the default there.

An alternative could be just not to test the hostname if the expectedBaseUrl is undefined.

Let's merge this PR as-is. If you feel strongly we can follow up with another PR.